src/context/TransactionsProvider/TransactionsProvider.tsx

```diff
@@ -155,6 +156,17 @@ export const TransactionsProvider: React.FC<TransactionsProviderProps> = ({ chil
     [],
   )
 
+  const maybeRefetchVotingPower = useCallback(
```

Comment:
worth only refetching on evm transactions? no applicable voting power changes related to other chains

Reply:
Yeah it's probably the best - the only way I can see a non-EVM Tx triggering a vp increase is a THOR swap to FOX, which should be pretty edge-case. If we want to be better at getting fresh data, then an interval refetch is probably a much saner option, so let's go with the least amount of requests at `<TransactionsProvider />` level for now - 5687b7af0432513d79f8df1d3e9e6d8517d30780